azure-quantum/azure/quantum/target/microsoft/elements/dft/libqcschema.py

```diff
@@ -0,0 +1,187 @@
+import json
```

Comment:
nit: could you please also add tests to `test_microsoft_elements_dft.py` to make sure these methods return expected format. Please, let me know if you won't be able to add tests and I could take this task on.